Wox/MainWindow.xaml

```diff
@@ -29,7 +29,17 @@
         <StackPanel Orientation="Vertical">
             <TextBox Style="{DynamicResource QueryBoxStyle}" Text="{Binding QueryText, Mode=TwoWay, UpdateSourceTrigger=PropertyChanged}" 
                      PreviewDragOver="TbQuery_OnPreviewDragOver" AllowDrop="True"
-                     x:Name="tbQuery" />
+                     x:Name="tbQuery">
+                <TextBox.ContextMenu>
+                    <ContextMenu>
+                        <MenuItem Command="ApplicationCommands.Cut"/>
+                        <MenuItem Command="ApplicationCommands.Copy"/>
+                        <MenuItem Command="ApplicationCommands.Paste"/>
+                        <Separator />
+                        <MenuItem Header="Settings" Click="menuSettings_Click" />
```

Comment:
I think it would be better to call `menuSettings_Click` as `ContextMenusForSettingsOnClick`.